### PR TITLE
walk: rename from llama

### PR DESCRIPTION
**Pull request by antonmedv**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/walk.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class Walk < Formula
+  desc "Terminal navigator"
+  homepage "https://github.com/antonmedv/walk"
+  url "https://github.com/antonmedv/walk/archive/refs/tags/v1.4.0.tar.gz"
+  sha256 "301f5dff2b8f27595a08a12626125d28313b5a92e97ccdaf4d0cb89111ebab9a"
```
Comment (chenrui333) [inline Formula/walk.rb:5]:
```suggestion
  sha256 "606568e098a91c7b0e694534bf0d268d92cc9ac19bce9e4671775837176eb9f5"
```

**Comment by antonmedv** [comment]
Sure, done.

File: Formula/walk.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/walk.rb:5]: what about the sha?

Comment (antonmedv) [reply Formula/walk.rb:5]: Trying to figure out. I updated in and pushed.

Comment (antonmedv) [reply Formula/walk.rb]: Done.

**Comment by antonmedv** [comment]
I don’t understand why go build is failing in tests.

File: Formula/walk.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+class Walk < Formula
+  desc "Terminal navigator"
+  homepage "https://github.com/antonmedv/walk"
+  url "https://github.com/antonmedv/walk/archive/refs/tags/v1.4.0.tar.gz"
+  sha256 "606568e098a91c7b0e694534bf0d268d92cc9ac19bce9e4671775837176eb9f5"
+  license "MIT"
+
```
Comment (chenrui333) [inline Formula/walk.rb:7]:
```suggestion
  depends_on "go" => :build

```

Comment (chenrui333) [reply Formula/walk.rb:7]: I am trying the build/test in my local.

**Comment by chenrui333** [comment]
actually it should be rename other new formula

**Comment by chenrui333** [comment]
failed with commit style 😱

**Comment by antonmedv** [comment]
Can’t understand why build is failing

**Comment by chenrui333** [comment]
> Can’t understand why build is failing

I will check in a bit (now we have sorted go 1.21.0)

**Review by branchv — APPROVED** [review]
Just needed a `depends_on "go"`, thanks!

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**